source/core/ut_suite_tag_filter.pkb

```diff
@@ -172,8 +173,17 @@ create or replace package body ut_suite_tag_filter is
   begin
     l_idx := a_postfix_exp.first;
     while ( l_idx is not null) loop
+      --If the token we got is a none or any keyword
+      if a_postfix_exp(l_idx) member of gc_reserved_tag_words then
+        
+        l_infix_stack.push( 
+          case 
+            when a_postfix_exp(l_idx) = 'none' then '('||a_tags_column_name||' is empty or '||a_tags_column_name||' is null)'
```

Comment:
What if someone will use a negated any/none?
Is it supported?
If it not supported, we should document it and handle it correctly.

Reply:
Yes its supported.
If we take the tests fron run_helper package we can call
```
select * from table(ut.run(a_tags => '!any'));

select * from table(ut.run(a_tags => 'none'));
```
which yields same results